### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by CalvinRodo** [appium/appium-desktop#460]
Hi,

 I work for a Canadian Government Department, we're required to use accessible software. 

We do a limited accessibility assessment as part of our software approval process. Appium-Desktop failed.

I've submitted the categories that failed as issues.

I've copied and pasted whole categories instead of entering each item. I've also included the other questions for that category in case you're interested.

I can try to get some more information on the failed categories.

I'll try and submit some pull requests if I get the time to work on this product. 

Categories Fixed
- [ ] #454 Issue with keyboard accessibility
- [ ] #454 Issue with Colour Accessibility
- [ ] #456 Issue with Cognitive Accessibility
- [ ] #457 Issue with the Accessibility of Screen Text
- [ ] #458 Icons, Images and Cursors
- [ ] #459 Accessibility Options

I'd be more than happy to try and get some more information on this stuff, I didn't do the assessment.

From the Report

>The checklist has its roots in “Section 508” of the United States Rehabilitation Act. Particularly, [Electronic and Information Technology Accessibility Standards (Section 508) Subpart B](http://www.access-board.gov/sec508/standards.htm#Subpart_b) 
Additionally, some guidance has been gained from the [IBM Software Accessibility Checklist - Version 3.5.1](http://www-03.ibm.com/able/guidelines/software/accesssoftware.html)  as well as the [Microsoft Accessibility Developer Center](http://msdn2.microsoft.com/en-us/accessibility/default.aspx)   and Microsoft Windows XP Logo Program Requirements, Section 3.0

**Comment by dpgraham** [comment]
Thanks for providing us with this info @CalvinRodo. I definitely would like to make Appium Desktop accessible. I don't know if I'll be able to prioritize time in the near future to work on this, but I'd be more than happy to accept pull requests on this.
